zcash_primitives/src/transaction/sighash_v5.rs

```diff
@@ -49,20 +54,54 @@ fn transparent_input_sigdigests<A: transparent::Authorization>(
         txid_digests.prevout_digest
     };
```

Comment:
Consider renaming to `prevouts_digest` (or to `prevouts_sig_digest` and add `_sig` to the other variable names), for consistency with ZIP 244.

Reply:
This value is also used in section T.2, and the naming is taken from there.